doc/release/upcoming_changes/15427.deprecation.rst

```diff
@@ -0,0 +1,10 @@
+Deprecation of probably unused C-API functions
+----------------------------------------------
+The following C-API functions are probably unused:
```

Comment:
unused and have been deprecated.

If you can say what the other API functions are in two words that would be nice